server/lib/wsapi/matchmaking.js

```diff
@@ -109,8 +110,12 @@ export class MatchmakingApi {
 
   async _onGameSetup(matchInfo, clients, players, setup = {}) {
     // TODO(2Pac): Select map intelligently based on user's preference
-    const mapInfo = (await getMapInfo(['4cd22c4f-2924-42f3-91ae-0e85ddaace3d']))[0]
+    const mapPool = await getCurrentMapPool(matchInfo.type)
+    if (!mapPool) {
+      throw new errors.NotFound('invalid map pool')
+    }
 
+    const mapInfo = (await getMapInfo(mapPool.maps))[0]
```

Comment:
The error below this (github won't let me add a comment there :( ) should probably also be 5xx, it implies some bug in the map selection process